Shivi_khanuja/Django/hello_world/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": false

```

Comment:
I agree on this change. The linter gets in the way more than it helps when you're learning how to program in the language.